source/PluginProcessor.cpp

```diff
@@ -2426,10 +2403,6 @@ void PolarDesignerAudioProcessor::setProxCompCoefficients (float distance)
 
 void PolarDesignerAudioProcessor::timerCallback()
 {
-    validateSampleRateAndBlockSize(); // Validate before operations
-
-    recomputeFilterCoefficientsIfNeeded();
```

Comment:
This is the culprit of the crash.

`timerCallback()` triggers recomputeFilterCoefficients regardless of whether the plugin is prepared or not.

This is not good. We should recompute filter coefficients when parameters change, not based on a timer.